source/custom/nitclk_service.custom.cpp

```diff
@@ -1,2 +1,61 @@
+#include <nitclk/nitclk_service.h>
+#include <stdexcept>
+
 namespace nitclk_grpc {
+    
+class DriverErrorException : public std::runtime_error{
```

Comment:
`DriverErrorException` and `CheckStatus` is typically used in other custom implementations for checking error from first library call. Since `status` of first library call represents `size` of buffer, we can't use `CheckStatus` for error validation. Since these are not used anywhere, you don't need to add these here.

Reply:
Done.